spec/unit/util/run_mode_spec.rb

```diff
@@ -10,14 +10,18 @@
     expect(@run_mode.run_dir).to eq('$vardir/run')
   end
 
+  it "has logdir depend on vardir" do
+    expect(@run_mode.log_dir).to eq('$vardir/log')
```

Comment:
@nwolfe `WindowsRunMode` still uses the base implementation of `RunMode#log_dir` and `RunMode#run_dir`:

    class WindowsRunMode < RunMode
      def conf_dir
        which_dir(File.join(windows_common_base("etc")), "~/.puppet")
      end

      def var_dir
        which_dir(File.join(windows_common_base("var")), "~/.puppet/var")
      end
      ...
    end